package.json

```diff
@@ -25,6 +25,7 @@
     "isomorphic-unfetch": "^2.0.0",
     "jsonwebtoken": "^8.2.2",
     "next": "^6.0.3",
+    "prisma": "^1.9.0",
```

Comment:
good call, I had it globally installed as per the prisma docs, but this is better